### ISSUE TITLE
update checker exception

### ISSUE THREAD

**Issue by LemADEC** [abauske/lance#2]
Whenever server boots, the update checker is reporting exception in console:

```
[11:08:01 WARN]: java.net.ConnectException: Connection timed out
[11:08:01 WARN]:        at java.net.PlainSocketImpl.socketConnect(Native Method)
[11:08:01 WARN]:        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
[11:08:01 WARN]:        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
[11:08:01 WARN]:        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
[11:08:01 WARN]:        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
[11:08:01 WARN]:        at java.net.Socket.connect(Socket.java:579)
[11:08:01 WARN]:        at java.net.Socket.connect(Socket.java:528)
[11:08:01 WARN]:        at sun.net.NetworkClient.doConnect(NetworkClient.java:180)
[11:08:01 WARN]:        at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
[11:08:01 WARN]:        at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
[11:08:01 WARN]:        at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
[11:08:01 WARN]:        at sun.net.www.http.HttpClient.New(HttpClient.java:308)
[11:08:01 WARN]:        at sun.net.www.http.HttpClient.New(HttpClient.java:326)
[11:08:01 WARN]:        at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:997)
[11:08:01 WARN]:        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:933)
[11:08:01 WARN]:        at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:851)
[11:08:01 WARN]:        at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1301)
[11:08:01 WARN]:        at java.net.URL.openStream(URL.java:1041)
[11:08:01 WARN]:        at knight37x.lance.Version$NewVersion.run(Version.java:73)
```

Please add an option to disable update checker for modpack authors.

**Comment by abauske** [comment]
Fixed!

**Comment by LemADEC** [comment]
Thanks!
When should we expect a new version?

**Comment by abauske** [comment]
What exactly do you mean by new version? Minecraft 1.8 or published fix?
The fix has been published already, see my website lance-mod.selfhost.eu new version is 2.6.2.1710!

**Comment by LemADEC** [comment]
Yes, I meant a published fix for 1.7.10, thanks!
